windows/src/README.md

```diff
@@ -1,85 +1,108 @@
 
   `make build` <-- builds Keyman Desktop and Keyman Developer
   `make clean` <-- remove temporary files and build artifacts
   `make release` <-- makes a release of all Keyman Windows projects
 
-Release build
--------------
+By default the version generated will be 10.0.700.0.
+
+### Release build
 
-1. Start a `VS2017 x64_x86 Cross Tools` command prompt.
-2. Run `make release` from the folder `windows/src` folder.
-3. Artifacts from a successful build will be placed in `windows/release` folder.
+To perform a release build, you will need to obtain valid certificates. A release build is
+unnecessary for local development.
+
+1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+2. Run `make release` from the **windows/src** folder.
+3. Artifacts from a successful build will be placed in **windows/release** folder.
 
 Note: by default, the version generated will be 10.0.700.0. You will not be able to
 install it over a later version of Keyman, and will need to uninstall and reinstall.
 
-Detailed Preqrequisite Notes
-============================
+## Installing Keyman
+
+These steps are only required the first time you install Keyman:
+1. Install release versions of Keyman 10.0 and Keyman Developer 10.0.
+    * Download the [alpha versions](https://keyman.com/beta/) and run the installers.
+2. Install the Keyman test certificates. Do the following for each KeymanTestCA cert in
+**windows/src/buildtools/certificates**:
+    1. Open the certificate and click 'Install certificate...' to open the Certificate Import Wizard.
+    2. Select either 'Current User' or 'Local Machine' for 'Store Location' depending on your preference and
+    click Next.
+    3. Select 'Place all certificates in the following store' and 'Browse...' to select
+    'Trusted Root Certification Authorities'. Click Next and finish the wizard.
 
-Visual Studio 2017 Community Edition setup requirements
--------------------------------------------------------
+To deploy a development build of Keyman and Keyman Developer,
+1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+2. Run `make signcode` from the **windows/src** folder.
+3. Ensure that Keyman and Keyman Developer are not running.
+4. Run `make install` in **windows/src/engine**, **windows/src/desktop** and **windows/src/developer**.
+
+## Detailed Prerequisite Notes
+
+### Visual Studio 2017 Community Edition setup requirements
 
 In Visual Studio 2017, you need to have the following components installed:
 * Universal Windows Platform development
+    * Desktop development with C++ > Windows 8.1 SDK and UCRT SDK
 * Desktop development with C++
-* 'Windows 8.1 SDK and UCRT SDK'
-* Universal CRT SDK
 
-* Two-space tab stops (spaces, not tabs). Some existing code has tabs rather than spaces
+Configure Visual Studio to use two-space tab stops:
+1. Open the options dialog: Tools > Options
+2. Navigate to Text Editor > All Languages > Tabs
+3. Change 'Tab size' to 2 and 'Indent size' to 2
```

Comment:
I believe the coding standard is to use spaces instead of tabs, so add a step 

4. Select "Insert spaces"